profile_service/src/main/kotlin/ru/zveron/exception/ProfileException.kt

```diff
@@ -0,0 +1,5 @@
+package ru.zveron.exception
+
+import java.lang.RuntimeException
+
+open class ProfileException(override val message: String?): RuntimeException(message)
```

Comment:
предлагаю так не делать, а брать по примеру как у меня:
```suggestion
import io.grpc.Status

open class ProfileException(status: Status, override val message: String?): RuntimeException(message)
```

Таким образом ты можешь в GrpcAdvice логику расписать типа такой: 
```
    @GrpcExceptionHandler(CategoryException::class)
    fun handleCategoryException(e: CategoryException): Status {
        logger.error { e.message }

        return e.status.withDescription(e.message)
    }
```

Reply:
то есть, прокидываются любые статусы, а не только захаркодженные Internal